src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers, numberOfPages, currentItems } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: string,
+  currentPage: number,
+  onPageChange: CallableFunction,
+  prevButton: CallableFunction,
+  nextButton: CallableFunction,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  prevButton,
+  nextButton,
+}) => {
+  const pagesCollection = getNumbers(1, numberOfPages(total, perPage));
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => prevButton()}
```

Comment:
```suggestion
            onClick={prevButton}
```
wrapper function is redundant here